changelog/28921.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
This should probably be called out as a `change` so it isn't lost in the bugs section of the changelog?

Also the message makes it sound like we never performed any certificate validation which isn't correct.